player/javascript.c

```diff
@@ -836,6 +836,59 @@ static void script_readdir(js_State *J, void *af)
     }
 }
 
+static void script_stat(js_State *J)
+{
+    const char *path = js_tostring(J, 1);
+
+    struct stat statbuf;
+    if (stat(path, &statbuf) != 0) {
+        push_failure(J, "Cannot stat path");
+        return;
+    }
+    // Clear last error
+    set_last_error(jctx(J), 0, NULL);
+
+    js_newobject(J);  // Result stat object
```

Comment:
There's a utility exactly for this kind of procedure - `push_nums_obj`, please use it, and then finally just add the two boolean values individually.

Reply:
Hmh. I wanted to keep it 1:1 with the Lua alternative, since both seem to have the same semantics.
Would there be performance corncerns or such?
If it's of importance, the repo members (should) have access to my branch.

Reply:
> I wanted to keep it 1:1 with the Lua

This use case is already different between lua and js.

> Would there be performance corncerns or such?

No, just less code for cases exactly like this. You could always add such utility at the lua code too ;)

> If it's of importance, the repo members (should) have access to my branch.

Not sure what this means or implies and why...

Reply:
The meaning/implication is that I made it this way, and since I'm currently not in the position to edit/git, the fastest way to change it is for you to just push the short form into my branch; I approve, I allow. I'd rather not mess with the github web editor.

Reply:
Not sure what "not in a position" means. It's your repo, so you can add commits, squash, force push, etc. It's not uncommon that PR reviews ask to modify some things...

Reply:
It means time and location deny me access to my computer. I just meant to say you have access to the branch. I like to do things effectively, and the faster it's fixed the faster the merge may happen - I "can't" (because I don't want to use the web editor) fix it now, so the next option is for someone else to do it. I'm sorry this became such a mess of a chain.

Reply:
It's not a mess. Take your time, and update it when you're in a position to. It's not a matter of life and death :)